src/lib/int.ts

```diff
@@ -364,9 +354,7 @@ class Int64 extends CircuitValue implements BalanceChange {
   // Overall, I think the existing implementation is the optimal one.
 
   constructor(magnitude: UInt64, sgn = Field.one) {
-    super();
-    this.magnitude = magnitude;
-    this.sgn = sgn;
+    super(magnitude, sgn);
```

Comment:
I left this constructor because it adds something nice -- a default value for one of the arguments